deploy/kubernetes/kiali.yaml

```diff
@@ -148,12 +149,6 @@ rules:
   - quotas
   - reportnothings
   - servicecontrolreports
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups: ["config.istio.io"]
-  resources:
```

Comment:
They were split because of the "delete" verb, we only delete a few objects among them. I think we should still require "delete" permission only for what we really need.

Reply:
Most of these objects are handlers and templates that depends on the rules (which is editable); so, I think it can be unified in the config.

Reply:
but are they supposed to be deleted when we delete a rule? I guess not, the delete command needs to be explicit on what we delete... so we'll never delete them.

I don't really like to ask for wider permissions than we actually need, just for having a prettier config file

Reply:
I'm not fully convinced that we cannot delete adapters/template.
Also, this is affecting the on-going work for update verb.
So, I suggest to maintain this change and we can refine continue discussing when the update is in place as I think we can delete/update adapters/templates inside rules.